hera_cal/io.py

```diff
@@ -740,6 +762,8 @@ def to_HERAData(input_data, filetype='miriad'):
     if isinstance(input_data, str):  # single visibility data path
         return HERAData(input_data, filetype=filetype)
     elif isinstance(input_data, HERAData):  # already a HERAData object
+        input_data._determine_blt_slicing()
```

Comment:
Why is this necessary? I think you might have explained it to me, but I honestly don't recall a scenario where a HERA data object would have been created and where this could have been run but wasn't.

Reply:
Yeah I agree: I think that's also just a bug w/ selecting on HERAData to begin with that this would fix..